recipe/meta.yaml

```diff
@@ -13,6 +14,9 @@ source:
 
 build:
   number: 100
```

Comment:
I mean we can limp along with this for a little while. We tried to pick some that worked now, but should be pretty stable for awhile (even if it is hideous). That way we have some time to work on something better for `conda` and/or `conda-build`.